### DISCUSSION TITLE
Can I use v-on:click in VPButton?

### DISCUSSION THREAD

**Discussion by xycabcd** [vuejs/vitepress#4445] [Q&A] [answered]
Is it possible to use VPButton with Vue?

**Comment by ms264556** [comment, marked as answer]
I use VPButton in several places on my vitepress site, with `@click` pointing to functions defined in the page's `<script setup>`. It works just fine.

When I was just using it on just one page, I included `import { VPButton } from 'vitepress/theme';` in my page's `<script setup>`. Now I have it added to my [enhanceapp](https://vitepress.dev/guide/extending-default-theme#registering-global-components): `app.component('VPButton', VPButton)` so I can easily use it everywhere.
